.gitignore

```diff
@@ -15,15 +15,12 @@
 vagrant_ansible_inventory_default
 
 # Drupal VM specific files.
-local.config.yml
-config.yml
-docker-compose.yml
-drupal.make.yml
-Vagrantfile.local
-examples/prod/inventory
-examples/prod/bootstrap/vars.yml
-scripts/
-drupal/
-
-# Some roles include 'config.yml' files.
-!provisioning/roles/**/config.yml
+/local.config.yml
```

Comment:
Since `local.config.yml` should only be used for a local instance configuration... maybe we leave that as globally ignored?